lr1110/wifi_helpers.patch

```diff
@@ -0,0 +1,13 @@
+--- wifi_helpers.c	2024-08-01 15:07:38
++++ wifi_helpers.c	2024-08-01 15:08:24
+@@ -158,8 +158,8 @@
+     max_nb_results = sizeof( wifi_results_mac_addr ) / sizeof( wifi_results_mac_addr[0] );
+     if( nb_results > max_nb_results )
+     {
+-        MW_DBG_TRACE_ERROR( "Wi-Fi scan result size exceeds %u (%u)\n", max_nb_results, nb_results );
+-        return false;
++        // Just use the number we received that can fit in our results array.
++        nb_results = max_nb_results;
```

Comment:
Why change this?

Reply:
The app returns an error if more than 5 results are detected, which sometimes occurs when there are many Wi-Fi access points nearby. This update retains all original functionality while preventing potential errors.

Reply:
That should go upstrem then

Reply:
Seeed Studio is not going to upstream this. It is caused by integrating their code with Tock OS.

Reply:
Then that's a bug that should be fixed, not just hacked around